### DISCUSSION TITLE
Dahua TiOC stopped working after upgrade to 0.12.0

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#6077] [Ask A Question] [answered]
I am hoping someone here can help me.

I have two Dahua TiOC clones (model IPC-T5241H-AS-PV with 3.6mm focal length from EmpireTech on Amazon). They have been working great. If Frigate detected a person in the zones I defined, my Home Assistant would trigger these camera's sirens. After upgrading to Frigate 0.12.0, the sirens on both cameras stopped working. I can't believe this is a coincidence. 

When I log into the cameras directly and try to manually play the alarm audio files, I get the following error message pop up in a box on the screen: ```"Failed to play audio or existing audio playback!"``` It doesn't matter whether I try the wav file that I've been using or one of the other ones--all of pre-installed files result in the same message. However, I can download these wav files from the cameras and play them on my laptop. So the files are definitely there.

The one thing I changed in these cameras when I upgraded to Frigate 0.12.0 was that I changed the audio encoding in the cameras. I believe they were on the default encoding, which is ```G.711A```. But the audio in the live view was choppy in webrtc and nonexistant in mse. So I set it to ```ACC``` following the documentation. That made the audio smooth in mse and nonexistant in webrtc, I don't know whether this setting change in the camera is connected to the inability to engage the sirens. I've tried changing it back to G.711A, but that does not restore my sirens.

Has anyone experienced anything like this? Any suggestions for what to try? Is there a way for me to reset the entire camera? I've looked at the Dahua API, but I don't understand the cameras enough to use the API commands to diagnose the issue.

I greatly appreciate any help. Many thanks in advance.

**Comment by NickM-27** [comment, marked as answer]
if I had to guess, the cameras support 2 way talk and the sirens are being blocked. You need to add `#backchannel=0` to the end of the streams in the go2rtc config

**Reply by Cold-Lemonade** [reply]
Thank you @NickM-27 

Is the "go2rtc config" the section of the config.yml file that start with ```go2rtc:```?

So I need to have it set as follows?

```
go2rtc:
  streams:
    front_camera:
      - rtsp://username:password@ip_address_of_camera:554/cam/realmonitor?channel=1&subtype=0#backchannel=0
```

**Reply by NickM-27** [reply]
yes that is correct

**Reply by Cold-Lemonade** [reply]
@NickM-27 You are my HERO! That fixed it! Thank you very much!
